doc/book/app_server/luajit_memprof.rst

```diff
@@ -61,38 +61,40 @@ Below is a chunk of simple Lua code named ``test.lua`` to illustrate this.
     end
     local stp, err = misc.memprof.stop()
 
-Starting profiler in Lua code:
+The Lua code for starting the profiler -- as in line 3 in the test.lua example above -- is:
 
 ..  code-block:: lua
 
     local str, err = misc.memprof.start(FILENAME)
 
-where ``FILENAME`` is a name of the binary file where profiling events are written.
+where ``FILENAME`` is the name of the binary file where profiling events are written.
 
-If it is not possible to open a file for writing or the profiler fails to start,
-``misc.memprof.start()`` returns ``nil`` on failure. Also, in this case
-the function returns an error message as the second result and
-a system-dependent error code as the third result.
-Otherwise, it returns ``true``.
+If the operation fails,
+for example if it is not possible to open a file for writing or if the profiler is already running,
+``misc.memprof.start()`` returns ``nil`` as the first result,
+an error-message string as the second result,
+and a system-dependent error code number as the third result.
+If the operation succeeds, ``misc.memprof.start()`` returns ``true``.
 
-Stopping profiler in Lua code:
+The Lua code for stopping the profiler -- as in line 18 in the test.lua example above -- is:
 
 ..  code-block:: lua
 
     local stp, err = misc.memprof.stop()
 
-If any error occurs at stopping the profiling
-(an error when the file descriptor is being closed) or during reporting,
-``misc.memprof.stop()`` returns ``nil``. Also, in this case
-the function returns an error message as the second result and
-a system-dependent error code as the third result.
-Otherwise, it returns ``true``.
+If the operation fails,
+for example if  there is an error when the file descriptor is being closed
+or if there is a failure during reporting,
+``misc.memprof.stop()`` returns ``nil`` as the first result,
+an error-message string as the second result,
+and a system-dependent error code number as the third result.
+If the operation succeeds, ``misc.memprof.start()`` returns ``true``.
```

Comment:
Should it be ``misc.memprof.stop()`` here?

Reply:
@pgulutzan Kindly help us resolve this issue. Is there a typo here? Thank you in advance.

Reply:
```suggestion
If the operation succeeds, ``misc.memprof.stop()`` returns ``true``.
```